tests/tizen_capi/unittest_tizen_capi.cc

```diff
@@ -488,6 +488,31 @@ file_cmp (const gchar * f1, const gchar * f2)
   return cmp;
 }
 
+/**
+ * @brief Wait until the change in pipeline status is done
+ * @return ML_ERROR_NONE success, ML_ERROR_UNKNOWN if failed, ML_ERROR_TIMED_OUT if timeout happens.
+ */
+int
```

Comment:
add static. no one is using it externally.

Reply:
Defining in /test/unittest_util.c looks good to prevent duplicated code for later.
I will change it.